rundeckapp/src/test/groovy/com/dtolabs/rundeck/server/plugins/RundeckPluginRegistrySpec.groovy

```diff
@@ -919,6 +922,103 @@ class RundeckPluginRegistrySpec extends Specification implements GrailsUnitTest
             'a:b:c' || 'a'
     }
 
+    def "StreamingLogReader with empty iterator should not fail to return a plugin"() {
+        when:
+        var pdir = File.createTempDir()
+        pdir.deleteOnExit()
+        RundeckPluginRegistry registry = new RundeckPluginRegistry()
+        registry.pluginDirectory = pdir
+        registry.metaClass.findBean = { String beanName -> return new TestStreamingLogReaderPlugin() }
+        registry.pluginRegistryMap = ["StreamingLogReader:mystreamingreader": "mystreamingreader"]
+        def actual = registry.loadBeanDescriptor("mystreamingreader","StreamingLogReader")
+        then:
+        actual
+
+    }
+
+    static class TestStreamingLogReaderPlugin implements StreamingLogReaderPlugin, Describable {
```

Comment:
Do we need this to be `Describable`?

Reply:
yes, otherwise it will return null which is not what we want

Reply:
Ohh, right, for the loadBeanDescription